django_admin_fast_search/admin.py

```diff
@@ -80,13 +80,14 @@ class GenericSearchFilter(admin.SimpleListFilter):
 
         template = "admin/custom_search_field.html"
 
-
         def lookups(self, request, model_admin):
             return (self.parameter_name, self.parameter_name),
 
         def queryset(self, request, queryset):
             if self.value():
-                kwargs = {"{}__search".format(self.parameter_name): f"*{self.value()}*"}
+                terms = self.value().strip().split(" ")
+                query = " ".join(["+" + term for term in terms])
+                kwargs = {"{}__search".format(self.parameter_name): f"{query}"}
```

Comment:
Should we use f-string formatting everywhere?

Reply:
Yes, it looks very odd. Not sure why I did this.